File: pascal_triangle/pascal_triangle.dart
import 'dart:io';

void main() {
  print('Enter size of the triangle: ');
  int size = int.parse(stdin.readLineSync()!);

  print('Enter character ASCII value for the triangle: ');
  String input = stdin.readLineSync()!;
  int charCode = input.codeUnitAt(0);

  for (int i = 0; i < size; i++) {
    for (int j = 0; j < size-1; j++) {
      stdout.write(' ');
    }
    for (int j = 0; j < i+1; j++) {
      stdout.write('${String.fromCharCode(charCode)} ');
    }
    stdout.writeln();
  }
}

File: pascal_triangle/pascal_triangle.c
#include<stdio.h>
int main(){
	int i, j, n, x;
	printf("Enter size of triangle: ");
	scanf("%d", &n);
	printf("Enter character value for triangle: ");
	scanf("%d", &x);
	for(i=0;i<=n;i++){
		for(j=0;j<=n-i;j++){
			printf(" ");
		}
		for(j=0;j<=i;j++){
			char ch = x;
			printf("%c ", ch);
		}
		printf("\n");
	}
	return 0;
}
